packages/agreement-email-sender/Dockerfile

```
@@ -1,6 +1,6 @@
 FROM node:20.14.0-slim@sha256:5e8ac65a0231d76a388683d07ca36a9769ab019a85d85169fe28e206f7a3208e as build 
 
-RUN corepack enable && corepack prepare pnpm@9.15.4 --activate
+RUN npm install -g corepack@0.31.0 && corepack enable && corepack prepare pnpm@9.15.2 --activate
```

```suggestion
RUN npm install -g corepack@0.31.0 && corepack enable
```

e stessa modifica in tutti gli altri punti